aws-ssm-ec2-proxy-command.sh

```diff
@@ -51,7 +51,8 @@ aws ssm send-command \
   --document-name 'AWS-RunShellScript' \
   --comment "Add an SSH public key to authorized_keys for 60 seconds" \
   --parameters commands="\"
-    cd ~${ssh_user}/.ssh || exit 1
+    [ -f ~/${ssh_user}/.ssh ] || mkdir -p ~/${ssh_user}/.ssh
```

Comment:
I think we can omit the test part and only do `mkdir -p ~/${ssh_user}/.ssh`, cause it checks for existence already.